core/field_angle.js

```diff
@@ -172,20 +172,32 @@ Blockly.FieldAngle.prototype.configure_ = function(config) {
   if (typeof clockwise == 'boolean') {
     this.clockwise_ = clockwise;
   }
-  var offset = Number(config['offset']);
-  if (!isNaN(offset)) {
-    this.offset_ = offset;
+
```

Comment:
If null -> do nothing -> use the value set on Blockly.FieldAngle, correct?

Worth a comment if so.

Reply:
Correct. I'll add that in a followup pr as well (though not worth cherry-picking).